spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/JobExecutionController.java

```diff
@@ -117,14 +119,39 @@ public JobExecutionController(TaskJobRepository repository) {
 	@RequestMapping(value = "/{id}", method = RequestMethod.GET, produces = "application/json")
 	@ResponseStatus(HttpStatus.OK)
 	public JobExecutionResource view(@PathVariable("id") long id) throws NoSuchJobExecutionException {
-		TaskJobExecution jobExecution = repository.getJobExecution(id);
+		TaskJobExecution jobExecution = taskJobService.getJobExecution(id);
 		if (jobExecution == null) {
 			throw new NoSuchJobExecutionException(String.format("No Job Execution with id of %d exits", id));
 		}
 		return jobAssembler.toResource(jobExecution);
 	}
 
 	/**
+	 * Stop Job Execution by the given jobExecutionId.
```

Comment:
probably want to document the parms (stop=true) in the .adoc and in the javadoc..